lib/stages/FEngine.hpp

```diff
@@ -104,6 +104,9 @@ class FEngine : public kotekan::Stage {
     // Dish reordering
     const std::vector<int> scatter_indices;
 
+    // Input buffers
+    const bool receive_chime;
```

Comment:
comment could be better. This flag indicates whether the input buffers use CHIME layout of CHORD layout, does it?